### PR TITLE
Add observability span context to logs

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
$Subject

Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1342

### CODE REVIEW

File: log-integration-tests/tests/resources/config/json/observability-project/Config.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[ballerina.observe]
+enabled=true
+provider="mock"
+
+[ballerina.log]
+format = "json"
+level = "DEBUG"
```
Comment (daneshk) [inline log-integration-tests/tests/resources/config/json/observability-project/Config.toml:7]:
```suggestion
format="json"
level="DEBUG"
```

File: log-integration-tests/tests/resources/config/logfmt/observability-project/Config.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[ballerina.observe]
+enabled=true
+provider="mock"
+
+[ballerina.log]
+level = "DEBUG"
```
Comment (daneshk) [inline log-integration-tests/tests/resources/config/logfmt/observability-project/Config.toml:6]:
```suggestion
level="DEBUG"
```

File: log-integration-tests/tests/resources/samples/observability-project/main.bal [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import ballerina/io;
```
Comment (daneshk) [inline log-integration-tests/tests/resources/samples/observability-project/main.bal:1]: shall we add licenses header here

File: log-integration-tests/tests/resources/config/json/observability-project/Config.toml (hunk 1 above)
Comment (MadhukaHarith92) [reply log-integration-tests/tests/resources/config/json/observability-project/Config.toml]: Fixed in 9d3f5b5

File: log-integration-tests/tests/resources/config/logfmt/observability-project/Config.toml (hunk 2 above)
Comment (MadhukaHarith92) [reply log-integration-tests/tests/resources/config/logfmt/observability-project/Config.toml]: Fixed in 9d3f5b5

File: log-integration-tests/tests/resources/samples/observability-project/main.bal (hunk 3 above)
Comment (MadhukaHarith92) [reply log-integration-tests/tests/resources/samples/observability-project/main.bal:17]: Added in 9d3f5b5

**Comment by MadhukaHarith92** [comment]
Temporarily disabled the observability test case in Windows due to https://github.com/ballerina-platform/ballerina-lang/issues/31168

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**